### PR TITLE
Add `shrey.dino.icu` and `codethetech.hackclub.com`

### PR DESCRIPTION
**Pull request by Shrey-Mehra**

### CODE REVIEW

**Comment by JosiasAurel** [comment]
please resolve conflicts

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2467,8 +2467,16 @@ hupy:
 
 
 
+codethetech:
+  ttl: 1
+  type: CNAME
+  values: codethetech.github.io
+  domain: www.codethetech.hackclub.com
+
```
Comment (grymmy) [inline hackclub.com.yaml:2475]: Please clean up whitespace to be consistent with the rest of the file.

File: dino.icu.yaml [hunk 2]
```diff
@@ -369,3 +369,9 @@ xn--c1h:
 xn--ct9h:
   type: CNAME
   value: b616c4f2-bfad-4b67-9c6d-ee6afc35cc0b.id.repl.co.
+
```
Comment (grymmy) [inline dino.icu.yaml:372]: Please make whitespacing consistent w/ rest of file.

**Review by grymmy — CHANGES_REQUESTED** [review]

File: dino.icu.yaml [hunk 3]
```diff
@@ -369,3 +369,8 @@ xn--c1h:
...
+  shrey:
+- ttl: 1
```
Comment (grymmy) [inline dino.icu.yaml:374]: The indentation of this section looks incorrect.

**Review by grymmy — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml [hunk 4]
```diff
@@ -2465,10 +2465,16 @@ hupy:
     type: CNAME
     value: hupy.github.io.
 
-
+codethetech:
+  ttl: 1
+  type: CNAME
+  values: codethetech.github.io
+  domain: codethetech.hackclub.com
```
Comment (Muirrum) [inline hackclub.com.yaml:2472]: Octodns expects a `value` key, here's a quick fix!

```suggestion
  value: codethetech.github.io
```

**Review by Muirrum — CHANGES_REQUESTED** [review]
Quick fix for DNS syntax.

Also -- can you edit this PR to include a quick description of what it's for?

File: hackclub.com.yaml [hunk 5]
```diff
@@ -2465,10 +2465,15 @@ hupy:
     type: CNAME
     value: hupy.github.io.
 
-
+codethetech:
+- ttl: 1
+  type: CNAME
+  value: codethetech.github.io
+
```
Comment (Muirrum) [inline hackclub.com.yaml:2472]: There doesn't seem to be a github pages here, and I think this site is out-of-scope for hackclub.com, @grymmy can you chime in on scope?

File: dino.icu.yaml [hunk 6]
```diff
@@ -369,3 +369,8 @@ xn--c1h:
 xn--ct9h:
   type: CNAME
   value: b616c4f2-bfad-4b67-9c6d-ee6afc35cc0b.id.repl.co.
+
+shrey: # a site to help many innovator by publishing many fun articles regarding technology
+  ttl: 1
+  type: CNAME
+  value: cname.vercel-dns.com.
```
Comment (Muirrum) [inline dino.icu.yaml:376]: Please put this description in the Pull Request description!

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by Shrey-Mehra** [comment]
Hey @reesericci i felt i should close this pull request because it's making me too much confusing regarding process . I want this domain but closing due to confusion

**Comment by reesericci** [comment]
Like I said - please feel free to open a new PR!

**Comment by Shrey-Mehra** [comment]
Sure thanks @reesericci

**Pull request closed without merging**